carbon-design-system/carbon-addons-iot-react
# fix(table): make table rows load more in skeleton state

Closes #3190

**Summary**

- In the [table row expansion](https://next.carbon-addons-iot-react.com/?path=/story/1-watson-iot-table-table--row-expansion-and-load-more) story, changed the row state while loading more so that it shows skeleton states rather than the loading button.

**Change List (commits, features, bugs, etc)**

- Add check for loading more state in TableBodyLoadMoreRow
- Add SkeletonText to each cell in the loading row
- Add unit test and a set of params/properties for it

**Acceptance Test (how to verify the PR)**

- Go to https://deploy-preview-3239--carbon-addons-iot-react.netlify.app?path=/story/1-watson-iot-table-table--row-expansion-and-load-more
- Click load more on table row
- Verify skeleton loading state

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Make sure the other Table stories with actions work as expected



**Things to look for during review**
